### ISSUE TITLE
feature request: include entire notes history with proposals

### ISSUE THREAD

**Issue by ctcpip** [tc39/proposals#509]
Problem: it is difficult (to varying degrees depending on circumstances) to track down the plenary notes for a given proposal.  for example, if you wanted a complete picture of the discussions for a proposal, you'd have to search the agendas repo and/or the notes repo using imperfect search functions. compounding this issue is that sometimes the proposals change names.

Solution: we already include the 'last presented' notes links in the tables; it is just as easy to, instead of replacing the 'last presented' link with the latest notes every time a proposal is presented, simply add a new link.  this adds no additional work than we are already doing today.

we should also endeavor to do this retroactively, but that shouldn't stop us from adopting this change right now for future notes links

#508 demonstrates this change in action.  see [rich diff](https://github.com/tc39/proposals/pull/508/files?short_path=af1d66e#diff-af1d66eb7dbbf6f66e871d26bcad07076a557256a957f558ca21e60924e2b0b7)

**Comment by ljharb** [comment]
The concept is excellent. It'd be ideal to find a way that won't shred the layout of the tables - do `<details>` links work inside a table cell?

**Comment by ctcpip** [comment]
shred the layout of the tables?  it doesn't affect the layout any more than how multiple champions on a proposal affects the table

**Comment by ctcpip** [comment]
we could also just have the links be [[0]](url),[[1]](url),[[2]](url), etc. as opposed to the entire month and year.  or shorthand them to [[2024-02]](url) style

**Comment by ljharb** [comment]
right but every proposal will likely have at least 4 notes links, whereas most proposals only have one champion.

**Comment by ctcpip** [comment]
> right but every proposal will likely have at least 4 notes links, whereas most proposals only have one champion.

_Temporal has entered the chat_

**Comment by ctcpip** [comment]
`February 2024`
`0,1,2,3,4,5,6`

looks how many mini-notes can fit into one mega-note!  🎶

**Comment by ljharb** [comment]
I think `2024.02, 2023.12, etc` might be clearer to read (also inside a sub)

**Comment by ctcpip** [comment]
sub makes it `10.5px` which is awful for a11y

**Comment by ljharb** [comment]
why is that awful?

(what's wrong with the dot? it's fine either way here, ofc)

**Comment by ctcpip** [comment]
are you asking why is a < 8 pt font size is poor a11y?

**Comment by ctcpip** [comment]
in any case, it appears GH renders the `<sub>` size differently in different places.. let me double check with a test md file

**Comment by ljharb** [comment]
Yes, I'm asking that, because that entirely depends on the eyes of the viewer and the quality and size of their screen. If someone can't see small text, they'll turn up the font size on their device and/or zoom in.

**Comment by ctcpip** [comment]
> Yes, I'm asking that, because that entirely depends on the eyes of the viewer and the quality and size of their screen. If someone can't see small text, they'll turn up the font size on their device and/or zoom in.

right, but it's not good to provide a usable font size and then <sub>add some ridiculously tiny text that they have to increase the zoom on just for the text, and thus impacts all other text, layout, etc.</sub>

<sub>in any case, it doesn't matter because the sub font on md is acceptable enough</sub>

**Comment by ctcpip** [comment]
tangentially, this is as good a time as any to complain about GH's max width, which is terribly small and reminds me I need to install the GH wide userstyle on my new workstation 😊

**Comment by michaelficarra** [comment]
https://github.com/tc39/dataset already includes all notes links for each proposal

**Comment by ljharb** [comment]
That sounds like a great way to populate the historical data here.

**Comment by mathiasbynens** [comment]
FWIW, I did this for my proposals ([example](https://github.com/tc39/proposal-regexp-v-flag#tc39-meeting-notes--slides)) and found it valuable.
